app/views/samples/index.haml

```diff
@@ -11,13 +11,16 @@
       =form_tag bulk_action_samples_path, :id => 'samples_form' , :method => :get do
         = cdx_table title: pluralize(@total, "sample") do |t|
           - t.actions do
-            = react_component "SampleTransfer", context: @navigation_context.to_hash, institutions: @institutions
-            =button_tag id: 'bulk_transfer', class: 'btn-link', name: 'bulk_action', value: 'transfer', type: 'button' do
-              .icon-send.btn-icon
-            =button_tag id: 'bulk_print', class: 'btn-link', name: 'bulk_action', value: 'print' do
-              .icon-print.btn-icon
-            =button_tag id: 'bulk_destroy', class: 'btn-link', name: 'bulk_action', value: 'destroy', data: { method: "delete", confirm: "You're about to permanently delete the selected samples. This action CANNOT be undone. Are you sure you want to proceed?" } do
-              .icon-trash.btn-icon
+            %div.table-info
+              = react_component "SampleTransfer", context: @navigation_context.to_hash, institutions: @institutions
+              =button_tag id: 'bulk_transfer', class: 'btn-link', name: 'bulk_action', value: 'transfer', type: 'button' do
+                .icon-send.btn-icon
+              =button_tag id: 'bulk_print', class: 'btn-link', name: 'bulk_action', value: 'print' do
+                .icon-print.btn-icon
+              =button_tag id: 'bulk_destroy', class: 'btn-link', name: 'bulk_action', value: 'destroy', data: { method: "delete", confirm: "You're about to permanently delete the selected samples. This action CANNOT be undone. Are you sure you want to proceed?" } do
+                .icon-trash.btn-icon
+              #notification.icon-error.icon-red.nodisplay
```

Comment:
`.nodisplay` means `display: hidden`. A better style would be `visibility: hidden` (`.hidden `). Space for such an error message should be reserved (even when invisible) so that it doesn't trigger a layout shift when it shows.